v2.0/src/Components/Pages/TeamsPage/TeamsPage.js

```diff
@@ -336,10 +336,14 @@ class TeamsPage extends React.Component {
       teamsSorted[i]["avrgActionsPerMember"] = avrg;
     }
 
+    //AD-HOC! This is for demo-ing JCAN so "Congregation Beth El" shows up first
     teamsSorted = teamsSorted.sort((a, b) => {
-      return b.avrgActionsPerMember - a.avrgActionsPerMember;
```

Comment:
the team is already coming in sorted so no work to be done.  so no need for the sort at all

Reply:
@Opoku-Agyemang I think there's a misunderstanding. Ellen wanted the teams sorted in _reverse_ (Z-A) for her demo - that would be the purpose of this temporary change.